v2/samples/keyvault/v1api/v1api20210401preview_vault.yaml

```diff
@@ -43,4 +44,3 @@ spec:
             - "get"
           storage:
             - "get"
-            
```

Comment:
Minor: New line

Reply:
The diff is interesting. I've _removed_ a _missing EOLN_ by adding one. 😉